e2etest/bgp_test.go

```diff
@@ -390,6 +390,130 @@ var _ = ginkgo.Describe("BGP", func() {
 			}),
 		)
 	})
+
+	ginkgo.Context("validate configuration changes", func() {
+		ginkgo.AfterEach(func() {
+			// Clean previous configuration.
+			err := updateConfigMap(cs, configFile{})
+			framework.ExpectNoError(err)
+		})
+
+		ginkgo.It("request a specific address pool for each Loadbalancer service", func() {
```

Comment:
Can you change the description to something like: it should work after subsequent configuration updates (or something alogn that line)?

Reply:
it sounds more like the context description, i wanted to be more specific here describing what exactly the test does.

Reply:
Yes but the point of this test is to try to change two different configurations I think. Otherwise, I think it's not totally clear.